src/elements/content-uploader/__tests__/ItemAction.test.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import noop from 'lodash/noop';
 import { shallow } from 'enzyme';
+import PlainButton from '../../../components/plain-button';
```

Comment:
Should we add test cases for null or undefined tooltip text?

Reply:
yes, i wanted to but i think i should avoid it for now since:
* i don't know how we would end up with an undefined tooltip - only if there was no status?
* should keep this PR just to the a11y bits. We've got a big pile of components to go through and update so we want to do a pass with all just the a11y stuff first.